PowerSimulationsPlots/src/call_plots.jl

```diff
@@ -252,53 +297,44 @@ function stack_plot(res::PSI.Results; kwargs...)
     save_fig = get(kwargs, :save, nothing)
     stacked_gen = get_stacked_generation_data(res)
     if isnothing(backend)
-        @info("no backend detected. Setting GR() as default.")
-        Plots.gr()
-    end
-    if backend == Plots.PlotlyJSBackend()
-        seriescolor = get(kwargs, :seriescolor, plotly_default)
-        plotly_stack_plots(res, seriescolor; kwargs...)
-        plotly_stack_gen(stacked_gen, seriescolor; kwargs...)
-    else
-        seriescolor = get(kwargs, :seriescolor, gr_default)
-        for name in string.(keys(res.variables))
-            variable_stack = get_stacked_plot_data(res, name)
-            p = RecipesBase.plot(variable_stack, name; seriescolor = seriescolor)
-            set_display && display(p)
-            !isnothing(save_fig) &&
-            Plots.savefig(p, joinpath(save_fig, "$(name)_Stack.png"))
-        end
-        p = RecipesBase.plot(stacked_gen; seriescolor = seriescolor)
-        set_display && display(p)
-        !isnothing(save_fig) && Plots.savefig(p, joinpath(save_fig, "Stack_Generation.png"))
-
+        throw(IS.ConflictingInputsError("No backend detected. Type gr() to set a backend."))
     end
+    _stack_plot_internal(res, stacked_gen, backend, save_fig, set_display; kwargs...)
 end
 
-function stack_plot(res::PSI.SimulationResults, variables::Array; kwargs...)
-    res_var = Dict()
-    for variable in variables
-        res_var[variable] = res.variables[variable]
-    end
-    results = PSI.OperationsProblemResults(
-        res_var,
-        res.total_cost,
-        res.optimizer_log,
-        res.time_stamp,
-    )
-    stack_plot(results; kwargs...)
+function _stack_plot_internal(
+    res::PSI.Results,
+    stack::StackedGeneration,
+    backend::Plots.PlotlyJSBackend,
+    save_fig::Any,
+    set_display::Bool;
+    kwargs...,
+)
+    seriescolor = get(kwargs, :seriescolor, plotly_default)
```

Comment:
Can you make all global constants capitalized so that it's obvious what they are?  (PLOTLY_DEFAULT)